### DISCUSSION TITLE
Lesson 6,  .env error

### DISCUSSION THREAD

**Discussion by JC-mb33** [Cyfrin/foundry-full-course-cu#1039] [Q&A] [answered]
Hi everyone,

I am having the following error when trying to run the script with the` .env` file.
`error: a value is required for '--fork-url <URL>' but none was supplied`

Here my code:
`$ forge script script/DeploySimpleStorage.s.sol --rpc-url $RPC_URL --broadcast  --private-key $PRIVATE_KEY`
I have checked for spaces, misspelling but I haven't found the issue. 
When forging the script without the` .env` file this works.

`.env` file is also correctly done according to the lesson. 
Perhaps I am missing something obvious?

**Comment by GenevaRoc** [comment]
You should be using this “$SEPOLIA_RPC_URL instead of just “$RPC_URL

**Reply by JC-mb33** [reply]
I have tried with Sepolia as well and is the same situation :/

actually when doing `source .env` I get the following now `bash: .env: No such file or directory`

**Comment by GenevaRoc** [comment]
did you run the command “source .env” first?

**Reply by JC-mb33** [reply]
Yes, I did

**Reply by DrBailey27** [reply, marked as answer]
Hello, is there a value for your SEPOLIA_RPC_URL in your .env file, if so then run the below on your terminal

    source .env
   echo $SEPOLIA_RPC_URL

you should get back your $SEPOLIA_RPC_URL value
Note that : you would have to run this in your terminal each time you turn on your system 
if this does not work
Ensure that .env is in your .gitignore file
 Check your spelling and ensure you wrote it the exact same way as Patrick did in your .env file and in your terminal
You could also ask chatgpt for help by sending your code and error message for help
I hope this works for you.

**Reply by JC-mb33** [reply]
Thanks man, the .env was missing on the .gitignore file.
Cheers!
